apps/applicant-tracking/src/components/ApplicantListItem/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { ApplicantResponse } from '@hack4impact-utk/internal-models';
+import { Chip, TableCell, TableRow } from '@mui/material';
+import LaunchIcon from '@mui/icons-material/Launch';
+import Button from '@mui/material/Button';
+
+//Each row should display the applicant name (first and last), status, NetID and application
+export default function ApplicantListItem(applicant: ApplicantResponse) {
+  return (
+    <TableRow>
+      <TableCell align={'left'} style={{ top: 57, minWidth: 170 }}>
```

Comment:
remove the style from each of these. I see what you're doing and why, but we can define the table component as `<Table sx={{ tableLayout: 'fixed' }}>...</Table>